### ISSUE TITLE
Crashes if # of items < list_limit

### ISSUE THREAD

**Issue by checkist84** [mickelson/attract#109]
For 1.5.0, if total number of items are less than 'list_limit' for current filter/list, the program crashes.

Seems 1.4.1 also has this problem...

**Comment by rogalian** [comment]
I don't have that problem here (and didn't with 1.4.1 either, my "Top 10" worked fine when it was still being populated with stats).

**Comment by checkist84** [comment]
If that's the case (some people does/doesn't have this problem), I think it might be helpful to add my system spec...

Tested on two Windows 7 (64bit) machines. Both 32/64bit executables had this problem.

**Comment by rogalian** [comment]
Same configuration here. Post the full filter?

**Comment by mickelson** [comment]
Hi I found this one, thanks for the report.

The crash happens when you start with a list that is less than the list_limit... so in your case rogalian the list would probably have been padded out still with games with stats at zero since the total list you were starting with was probably more than 10.

There's also a really dumb bug for emulator configurations that I've just fixed.  I'm not sure how I didn't stumble over it earlier.  I'm going to put together a 1.5.1 release to address these issues.
